src/main/java/com/github/paweladamski/httpclientmock/HttpClientMockBuilder.java

```diff
@@ -110,6 +137,17 @@ public HttpClientMockBuilder withBody(Matcher<String> matcher) {
     return this;
   }
 
+  /**
+   * Adds body condition. Request body must contain the given URL-encoded form parameters (typically used in POST requests). Alternatively, parameters may be specified individually using {@link #withFormParameter(String, Matcher)}.
+   * 
+   * @param parameters the parameters
+   * @return condition builder
+   */
+  public HttpClientMockBuilder withBody(Map<String, Matcher<String>> parameters) {
```

Comment:
I think name of this method is a bit confusing. If a method for setting a single form parameter is called `withFormParameter` than I would expect  that method for setting a collection of params would be `withFormParameters`

Reply:
Me too. Technically, form parameters are located inside of the request body. That's why I created the method. Changing the name to `withFormParameters` is OK, I think.